Snippets/ServiceFabricPersistence/ServiceFabricPersistence_1/OutboxConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using NServiceBus;
+
+class OutboxConfiguration
+{
+    void OutboxConfigurationUsage(EndpointConfiguration endpointConfiguration)
+    {
+        #region ServiceFabricPersistenceOutboxConfiguration
+
+        var outboxSettings = endpointConfiguration.EnableOutbox();
+        outboxSettings.SetTimeToKeepDeduplicationData(TimeSpan.FromDays(1));
```

Comment:
if extension methods clash they can always be qualified by calling the class explicitly.